frontend/types/types.ts

```diff
@@ -92,6 +92,13 @@ export interface FileInterface {
 }
 
 export interface AvScanResult {
+  artefactKind: ArtefactKindKeys
```

Comment:
I think we should also add the discriminating union here so that we don't accidentally type files with properties that only used by images